Features/Context/SubContext/Authentication.php

```diff
@@ -73,7 +75,8 @@ public function iShouldBeLoggedIn()
     {
         $this->spin(
             function () {
-                $logoutElement = $this->getSession()->getPage()->find('css', '.ez-logout');
+                $this->findWithWait('.ez-user-profile')->click();
```

Comment:
to check if a user is logged in,  I think it's enough to find the `.ez-user-profile` element